src/checkbox/checkbox.js

```diff
@@ -15,255 +15,148 @@
  * limitations under the License.
  */
 
-(function() {
-  'use strict';
+// Numeric constants used in this component.
+const numbers_ = Object.freeze({
+  TINY_TIMEOUT: 0.001
+});
+
+// CSS classes used in this component.
+const classes_ = Object.freeze({
```

Comment:
Assuming it is done in the constructor that is just the simplest method. Also, introducing `.prototype.whatever = {}` after the class is defined then introduces problems with re-usability. It then adds more work on people using individual bits to pull out just what they need.